cmd/codecovgen/main.go

```diff
@@ -117,7 +117,7 @@ var (
 	// validFlagRegexp is a regular expression to validate codecov flags.
 	// It is mentioned here: https://docs.codecov.com/docs/flags
 	// It's unclear if components fit this pattern, but they seem to work with it.
-	validFlagRegexp = regexp.MustCompile(`^[\w\.\-]{1,45}$`)
+	validFlagRegexp = regexp.MustCompile(`^[\w.\-]{1,45}$`)
```

Comment:
Why is this a simplification? Doesn't this change the exact dot match to any character?